CHANGELOG.md

```diff
@@ -3,11 +3,164 @@
 [Releases](https://github.com/NetApp/harvest/releases)
 
 ## 22.02.0 / 
+<!-- git log --no-decorate --no-merges --cherry-pick --right-only --oneline origin/release/21.11.0...origin/release/22.02.0 -->
 
-**IMPORTANT** Admin node certificate file location changed. Certificate files have been consolidated into the cert directory. You need to move your admin-cert.pem and admin-key.pem files to the cert directory.
+:boom: Highlights of this major release include:
+
+- Continued progress on the ONTAP REST config collector. Most of the template changes are in place and we're working on closing the gaps between ZAPI and REST. We've made lots of improvements to the REST collector and the 13 REST templates in this release. The REST collector  should be considered early-access as we continue to improve it. If you try it out or have any feedback, let us know on Slack or [GitHub](https://github.com/NetApp/harvest/discussions). :book: You can find more information about when you should switch from ZAPI to REST, what versions of ONTAP are supported by Harvest's REST collector, and how to fill ONTAP gaps between REST and ZAPI documented [here](https://github.com/NetApp/harvest/blob/main/docs/architecture/rest-collector.md)
```

Comment:
We've made lots of improvements to the REST collector and **enhanced/added** 13 REST templates in this release